runtime/compilers/rillv1/parser.go

```diff
@@ -178,6 +178,9 @@ type Parser struct {
 	Resources map[ResourceName]*Resource
 	Errors    []*runtimev1.ParseError
 
+	// Mapping of local data files to resource paths that depend on them
+	LocalDataToResourcepath map[string]map[string]any
```

Comment:
See other comment, it would be very nice if this could be kept as internal state

Reply:
Yes that makes sense.